guides/relay/mutations.md

```diff
@@ -41,10 +45,9 @@ For Relay-compliant GraphQL, a mutation field must comply to a strict API. `Grap
 After defining a mutation (see below), add it to your mutation type:
 
 ```ruby
-MutationType = GraphQL::ObjectType.define do
-  name "Mutation"
+class MutationType < GraphQL::Schema::Object
   # Add the mutation's derived field to the mutation type
-  field :addComment, field: AddCommentMutation.field
+  field :add_comment, field: AddCommentMutation.field
```

Comment:
I think `mutation:` should be used instead of `field:`

Reply:
No, a mutation type is just an object type and so it defines fields. For example, check out how the mutation is used in the specs: https://github.com/rmosolgo/graphql-ruby/blob/f77dd391f3e8a46b094808820c15e858a3ef7bd1/spec/support/star_wars/schema.rb#L334

Reply:
Sorry, I misread your comment. You're right, the argument should be mutation rather than field.

Reply:
There's some confusion here because of `GraphQL::Relay::Mutation` (old) vs `GraphQL::Schema::Mutation` (new) 

For the _old_ one, you can do `field: OldMutation.field`. For the _new_ one, you can do `mutation: NewMutation`. 

This guide starts with a note to direct people to the new mutation docs. And in a month or so, I'll probably remove this one altogether, so it's ok with me to leave it for now.